src/main/java/net/minecraftforge/fml/common/Loader.java

```diff
@@ -941,20 +941,22 @@ boolean checkRemoteModList(Map<String, String> modList, Side side)
     /**
      * Fire a FMLMissingMappingsEvent to let mods determine how blocks/items defined in the world
      * save, but missing from the runtime, are to be handled.
+     * @param missingMultiparts 
      *
      * @param missing Map containing missing names with their associated id, blocks need to come before items for remapping.
      * @param isLocalWorld Whether this is executing for a world load (local/server) or a client.
+     * @param remapMultiparts 
      * @param gameData GameData instance where the new map's config is to be loaded into.
      * @return List with the mapping results.
      */
-    public List<String> fireMissingMappingEvent(Map<ResourceLocation, Integer> missingBlocks, Map<ResourceLocation, Integer> missingItems, boolean isLocalWorld, Map<ResourceLocation, Integer[]> remapBlocks, Map<ResourceLocation, Integer[]> remapItems)
+    public List<String> fireMissingMappingEvent(Map<ResourceLocation, Integer> missingBlocks, Map<ResourceLocation, Integer> missingItems, Map<ResourceLocation, Integer> missingMultiparts, boolean isLocalWorld, Map<ResourceLocation, Integer[]> remapBlocks, Map<ResourceLocation, Integer[]> remapItems, Map<ResourceLocation, Integer[]> remapMultiparts)
```

Comment:
We really need to move this to a map or something so we can add everything we need without changing the signature.... @cpw ?